gradle/libs.versions.toml

```diff
@@ -36,6 +36,7 @@ rabbitMq = "5.25.0"
 versionsPlugin = "0.52.0"
 
 # Gradle dependencies
+aedile = "2.0.3"
```

Comment:
I am sceptic whether this functionality warrants adding a full-blown caching library. I would prefer a simple manual implementation, so that we know what is going on behind the scenes.

Reply:
The "simple" manual implementation would still have to deal with time-based eviction and thread/coroutine-safety,  and have to prevent that the cache uses too much memory. What's the benefit over using a library that's proven to work?

Reply:
On the other hand, using a well known library (which [caffeine](https://github.com/ben-manes/caffeine) is) might solve some issues in edge cases which are currently unknown to us.
I think in this case it's still ok to use a library.